article/app/views/fragments/articleBody.scala.html

```diff
@@ -42,38 +42,16 @@
                 @fragments.guBand()
             }
 
-            @if(amp) {
-                @fragments.headTonal(article, model, isPaidContent, amp = amp)
-
-                @if(article.tags.isFeature && article.elements.hasShowcaseMainElement) {
-                    @fragments.mainMedia(article, amp)
-                }
-            } else {
-                <div class="hide-on-mobile">
-                    @fragments.headTonal(article, model, isPaidContent, amp = amp)
-                </div>
+            @fragments.mainMedia(article, amp)
```

Comment:
This file is still being used when we render an article as `.json`. What that is used for I am not sure!